app/forms/application_form.rb

```diff
@@ -12,4 +11,22 @@ def self.model_name
     # This allows Rails magic such as route paths.
     ActiveModel::Name.new(self, nil, to_s.delete_suffix(FORM_CLASS_SUFFIX))
   end
+
+  def self.accepts_nested_attributes_for(*models, **_options)
```

Comment:
Here's the home-grown nested attrs impl for our ActiveModel-based forms. No options (such as `allow_destroy` have been added yet).

Reply:
Can you narrate this method a bit more in comments?

Reply:
Will do.